sru-scripts/manual/oracle-sru

```diff
@@ -20,10 +20,25 @@ create_setup_proposed_script
 
 echo "### BEGIN $UBUNTU_RELEASE"
 
-# TODO: Use the Oracle CLI
-echo "Please manually launch a $UBUNTU_RELEASE instance in Oracle and enter its IP address"
-echo "(Don't forget to add your SSH public key to the instance when you launch it.)"
-read -p "$UBUNTU_RELEASE IP address: " VM_IP
+declare -A releases
+releases=( ["xenial"]="16.04" ["bionic"]="18.04" ["focal"]="20.04" )
+
+IMAGE_ID=$(oci compute image list --operating-system="Canonical Ubuntu" --operating-system-version="${releases[${UBUNTU_RELEASE}]}" --sort-by="TIMECREATED" --limit=1 2>/dev/null | jq -r '.data[0]["id"]')
+VCN_ID=$(oci network vcn list | jq -r '.data[0]["id"]')
+SUBNET_INFO=$(oci network subnet list --vcn-id="${VCN_ID}" | jq '.data[0] | {id: .id, "availability-domain": ."availability-domain"}')
+SUBNET_ID=$(echo ${SUBNET_INFO} | jq -r '.id')
+AVAILABILITY_DOMAIN=$(echo ${SUBNET_INFO} | jq -r '."availability-domain"')
+
+INSTANCE_ID=$(oci compute instance launch --availability-domain="${AVAILABILITY_DOMAIN}" --shape="VM.Standard1.1" --image-id="${IMAGE_ID}" --subnet-id="${SUBNET_ID}" --ssh-authorized-keys-file="${SSH_KEY}" | jq -r '.data["id"]')
```

Comment:
The annoying thing here is that return code of the `oci ... | jq ...` command is the one of `jq`, so we miss the return code of `oci` and fail to exit early from the script in case of errors. The easiest way out is to run the script with:
```
set -ex -o pipefail
```
But we have another problem: if the launch command succeeds but `jq` fails then we fail to get the `INSTANCE_ID` and thus fail to delete the instance. I think reason number 1 why `jq` can fail is that it's missing. Adding a
```
hash jq
```
line at the beginning of the script will make it exit early if the shell can't find `jq`. We rely on `set -e` for this, which learned not to like very much, but I think it's OK in this case.